Arrowgene.Ddon.GameServer/Handler/EntryBoardEntryBoardItemCreateHandler.cs

```diff
@@ -0,0 +1,71 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.GameServer.Characters;
+using Arrowgene.Ddon.GameServer.Dump;
+using Arrowgene.Ddon.GameServer.Utils;
+using Arrowgene.Ddon.Server;
+using Arrowgene.Ddon.Server.Network;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Network;
+using Arrowgene.Logging;
+using System;
+
+namespace Arrowgene.Ddon.GameServer.Handler
+{
+    public class EntryBoardEntryBoardItemCreateHandler : GameRequestPacketHandler<C2SEntryBoardEntryBoardItemCreateReq, S2CEntryBoardEntryBoardItemCreateRes>
+    {
+        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(EntryBoardEntryBoardItemCreateHandler));
+
+        public EntryBoardEntryBoardItemCreateHandler(DdonGameServer server) : base(server)
+        {
+        }
+
+        public override S2CEntryBoardEntryBoardItemCreateRes Handle(GameClient client, C2SEntryBoardEntryBoardItemCreateReq request)
+        {
+            // var pcap = new S2CEntryBoardEntryBoardItemCreateRes.Serializer().Read(GameFull.Dump_710.AsBuffer());
+            var result = new S2CEntryBoardEntryBoardItemCreateRes()
+            {
+                BoardId = request.BoardId,
+            };
+
+            result.EntryItem.Param = request.CreateParam;
+            result.EntryItem.PartyLeaderCharacterId = client.Character.CharacterId;
+            result.EntryItem.TimeOut = 3600;
+            result.EntryItem.Id = Random.Shared.NextU32();
```

Comment:
I suppose this can work. Assuming the server runs long enough without a restart, we need to make sure these ID's are still not in use. I guess another map to keep track of in this ExmManager.

Reply:
Done.